specs/schemas/workspace.json

```diff
@@ -1,10 +1,14 @@
 {
-  "$schema": "<http://json-schema.org/draft-07/schema#>",
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
```suggestion
  "$schema": "http://json-schema.org/draft-12/schema",
```

Reply:
The URL you're suggesting doesn't exist.

Reply:
thanks for pointing that out, updated my initial comment.

Reply:
Is the idea to use the latest version here? Some tools don't support later versions of the JSON schema spec